README.md

```diff
@@ -403,6 +403,30 @@ type UseName = {
 };
 ```
 
+### hasVerifiedAttestations
+
+```tsx
+import { base } from 'viem/chains';
+import { hasVerifiedAttestations } from '@coinbase/onchainkit';
+
+const hasVerifiedAttestation = await hasVerifiedAttestations(
```

Comment:
Is `has` the right wording for an API 🤔

Also can we we document what't the return type.

Reply:
ok love that!